modules/effects/src/main/scala/dev/profunktor/redis4cats/algebra/server.scala

```diff
@@ -16,9 +16,10 @@
 
 package dev.profunktor.redis4cats.algebra
 
-trait ServerCommands[F[_]] extends Flush[F]
+trait ServerCommands[F[_], K] extends Flush[F, K]
 
-trait Flush[F[_]] {
+trait Flush[F[_], K] {
+  def keys(pattern: K): F[List[K]]
```

Comment:
For consistency could you please rename `pattern` to `key`?

Reply:
Done